CHANGELOG.md

```diff
@@ -1,13 +1,80 @@
+## Fleet 4.55.0 (Aug 8, 2024)
+
+**NOTE:** Beginning with v4.55.0, Fleet no longer supports MySQL 5.7 because it has reached [end of life](https://mattermost.com/blog/mysql-5-7-reached-eol-upgrade-to-mysql-8-x-today/#:~:text=In%20October%202023%2C%20MySQL%205.7,to%20upgrade%20to%20MySQL%208.). The minimum version supported is MySQL 8.0.36.
+
+### Endpoint Operations
+
+- Added support for generating `fleetd` packages for Linux ARM64.
+- Added new `fleetctl package` --arch flag.
+- Updated `fleetctl package` command to remove the `--version` flag. The version of the package can be controlled by `--orbit-channel` flag.
```

Comment:
Just realized these are technically mdm since Dante did that but unsure if they should stay here or move to mdm section since they aren't necessarily MDM

Reply:
I'm organizing changes by feature category vs. team category now that we're doing a fair amount of cross work. I try to following the feature category defined in https://github.com/fleetdm/fleet/blob/main/handbook/company/pricing-features-table.yml.